src/schema/schema_create.c

```diff
@@ -985,9 +1097,23 @@ __schema_create(WT_SESSION_IMPL *session, const char *uri, const char *config)
      * back it all out.
      */
     WT_RET(__wt_meta_track_on(session));
-    if (import)
+    if (import) {
         F_SET(session, WT_SESSION_IMPORT);
 
+        if (__wt_config_getones(session, config, "import.metadata_file", &cval) == 0 &&
+          cval.len != 0 && (cval.type == WT_CONFIG_ITEM_STRING || cval.type == WT_CONFIG_ITEM_ID)) {
+            WT_ERR(__wt_strndup(session, cval.str, cval.len, &export_file));
+            import_list.uri = uri;
+
+            /* Get suffix of the URI. */
+            import_list.uri_suffix = strchr(uri, ':');
+            WT_ASSERT(session, import_list.uri_suffix != NULL && import_list.uri_suffix[1] != '\0');
```

Comment:
Do we need this assert here? From my understanding L1091 prefix matches for `file:` or `table:` anyways. I am thinking we can utilise WT_PREFIX_SKIP_REQUIRED here instead

Reply:
L1091 checks if the `uri` starts with `file:` or `table:`, it is prefix validation. Here we check that _suffix_ is not empty, i.e. assert will fail if `uri="table:"` without the actual table name.